.github/workflows/build-ffmpeg.yml

```diff
@@ -48,3 +48,120 @@ jobs:
         with:
           name: output
           path: output/
+
+  build-aarch64-stage-1:
+    name: build (ubuntu-latest, aarch64) stage-1
+    runs-on: ubuntu-latest
+    env:
+      pybin: /opt/python/cp38-cp38/bin/
+      vendor: /tmp/vendor
+      img: quay.io/pypa/manylinux2014_aarch64
+      stage: 1
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - uses: actions/cache@v2
+        id: libs-cache
+        with:
+          path: vendor
+          key: ${{ runner.os }}-aarch64-libs-cache-${{ env.stage }}-${{ hashFiles('**/build-ffmpeg.py') }}
+      - name: Build FFmpeg dependencies
+        if: ${{ steps.libs-cache.outputs.cache-hit != 'true' }}
+        run: |
+          docker run -v $PWD:/project:rw --workdir=/project ${{ env.img }} bash -exc '
+            yum -y install gperf libuuid-devel zlib-devel;
```

Comment:
The Python script already invokes yum, do we need it here too?

Reply:
Yum requires ```liblzma.so``` which is already present in the system. We are building a different version of ```liblzma.so``` for our own purpose and adding its path to LD_LIBRARY_PATH. If we allow script to execute ```yum```, ```yum``` will detect our built ```liblzma.so``` which is incompatible. causing installation failure.

To avoid this scenario we need to execute ```yum``` before adding ```liblzma.so```'s path to LD_LIBRARY_PATH.

Reply:
I don't think you need to touch LD_LIBRARY_PATH at all for stage1 and stage2, I think it's only actually needed to *test* the wheel which is built at the very end.

Reply:
Please take a look at below code:
```
Package(
            name="gnutls",
            requires=["nettle", "unistring", "zlib"],
            source_url="https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.16.tar.xz",
            build_arguments=[
                "--disable-cxx",
                "--disable-doc",
                "--disable-libdane",
                "--disable-nls",
                "--disable-tests",
                "--disable-tools",
                "--with-included-libtasn1",
                "--without-p11-kit",
            ],
        ),
```

```gnutls``` requires ```nettle```, ```unistring``` and ```zlib```. And these are being install on /tmp/vendor. So, to make it visible to build tool and use as a dependency we need to add **/tmp/vendor** to **LD_LIBRARY_PATH**. Please correct me if I am wrong. 

However, I will do a side experiment on your suggestion and share you. :)

Reply:
`LD_LIBRARY_PATH` is not used when *linking* (we already handle this by setting `LDFLAGS`), it's used when a executable *loads* a shared library. The only case I can think of is when at the very end we test a wheel which is linked against ffmpeg (and transitively all its dependencies).

Reply:
Removing LD_LIBRARY_PATH causes below failure-
Commit: https://github.com/odidev/pyav-ffmpeg/commit/4706d0d9e807bd10b0c1a3721e0e5d7364ff9edd
Logs: https://github.com/odidev/pyav-ffmpeg/runs/5289695969?check_suite_focus=true#step:6:10805

Reply:
Adding ```LDFLAGS``` should work but not sure why it is failing in above experiment.

Reply:
It is failing while running ```configure``` script and I think ```LDFLAGS``` will not help here.

Reply:
This is very weird, but I can definitely reproduce it, my test build failed in the same way. I'll try and debug it. This really surprises me because the test in the `gnutls`  `configure` script uses `AC_CHECK_FUNCS` which only relies on the ability to *link* a program, not execute it.